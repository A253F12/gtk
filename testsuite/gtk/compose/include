include "testsuite/gtk/compose/included"   # see if this works

<Multi_key> <s> <e> <q> : "!"
