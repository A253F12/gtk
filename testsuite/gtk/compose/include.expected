# n_seqs: 2
# max_seq_len: 4
<Uff20> <U73> <U61> <U73> : "\"\\"
<Uff20> <U73> <U65> <U71> : "!" # U21
